capabilities/version-control.md

```diff
@@ -1,28 +1,109 @@
 # [Version Control](https://dora.dev/devops-capabilities/technical/version-control/)
 
-Version control allows teams to revert changes back to a working state. When this is fast an easy to accomplish, teams can move forward with less fear of breaking the system. Technologies like Git and SVN have made version control for code popular. However, this DORA Capability applies all the same to configuration, data, infrastructure, and documentation. Furthermore, version control is foundational for enabling automation, ensuring disaster recovery, and facilitating continuous integration.
+Comprehensive version control ensures reproducibility and traceability across the system, enabling teams to provision production-like environments, track changes, and respond quickly to evolving needs. This capability enhances disaster recovery, auditability, and quality by allowing teams to restore services efficiently, demonstrate process integrity, and receive rapid feedback on changes. When teams can scale their environments and roll back to verified working states, when necessary, they can support capacity management and defect response.
```

Comment:
I added "across the system" to the first line here, but is that needed/accurate? My goal was to clarify "what" was reproducible and traceable.

Reply:
That's fine. I wouldn't call it strictly necessary, but it's not wrong.